pages/home.jsx

```diff
@@ -101,13 +105,15 @@ var BlogSection = React.createClass({
     }
   },
   componentDidMount: function() {
-    var self = this;
-    blogFeedLoader(this, window, "https://blog.webmaker.org/tag/teachtheweb/feed", function(data) {
-      self.setState({
-        featuredPostData: data.featuredPostData,
-        latestPostsData: data.latestPostsData
+    this.props.loadBlogPosts(function(data) {
+      if (!this.isMounted()) {
+        return;
+      }
+      this.setState({
+        featuredPostData: data.featuredPosts,
+        latestPostsData: data.latestPosts
```

Comment:
I decided to change the data's properties to not include the word `Data` at the end, since it seemed redundant... You can feel free to change the naming of your view props too if you want, but it's up to you.